scripts/qa_crowbarsetup.sh

```diff
@@ -314,12 +314,12 @@ function add_sdk_repo
     fi
 }
 
-function add_sap_repo
+function add_continuousdelivery_repo
 {
     local stage=
     [[ $TESTHEAD ]] && stage=":/Staging"
     # priority required to overwrite the default cloud6 packages
-    $zypper ar -p 92 -f http://$susedownload/ibs/Devel:/Cloud:/6:/SAP${stage}/SLE_12_SP1/ dc6sap
+    $zypper ar -p 92 -f http://$susedownload/ibs/Devel:/Cloud:/6:/SAP${stage}/SLE_12_SP1/ dc6cd
```

Comment:
This has SAP in the URL so it still seems like a SAP repo to me

Reply:
@cmurphy yeah, but can't rename the SAP project easily. that was a misnomer when it was set up